pkgs/data/themes/orchis-theme/default.nix

```diff
@@ -6,6 +6,7 @@
 , gtk-engine-murrine
 , sassc
 , tweaks ? [ ] # can be "solid" "compact" "black" "primary"
+, wallpapers ? false
```

Comment:
Could we name this something like `withWallpapers` instead? It's a bit more descriptive of the options type.